src/EditorFeatures/CSharpTest/Diagnostics/MakeMethodAsynchronous/MakeMethodAsynchronousTests.cs

```diff
@@ -1279,7 +1279,8 @@ void M()
         public async Task MethodWithNullableReturn()
         {
             await TestInRegularAndScriptAsync(
-@"using System.Threading.Tasks;
+@"#nullable enable
```

Comment:
i'd somewhat prefer test duplciation, showing what you get with/without this attribute.

Reply:
basically, i expect us to break things in the future, and i think having the variety of tests will help us catch things early.

Reply:
This test was just outright wrong: it was using ? in cases where we didn't enable nullable.

Reply:
(and it's potentially bad that you had a ? where it was illegal and we dropped it, but then that requires some extra handling to differentiate between "this type symbol was illegal but you want it anyways" and "you want us to make things legal". If we get feedback I'll be sympathetic about it, but when *that's* our biggest problem I'll be happy!)